CURRENT_VERSION

```diff
@@ -1 +1 @@
-3.2.0
+4.0.0
```

Comment:
May want to fix the newline issue here; For example, I don't know if its absence breaks any text scraping we do with this file.

Reply:
It failed the build when I had a newline there.